utils/uniset/src/uniset.rs

```diff
@@ -175,57 +175,57 @@ impl<'data> CodePointSet<'data> {
     /// # Examples
     ///
     /// ```
-    /// use icu_uniset::CodePointSet;
+    /// use icu_uniset::CodePointInversionList;
     /// use icu_uniset::CodePointSetError;
     /// use zerovec::ZeroVec;
     ///
     /// use std::vec::Vec;
     ///
-    /// fn inv_list_to_owned_codepointset(inv_list: &[u32]) -> CodePointSet {
-    ///     CodePointSet::clone_from_inversion_list_slice(inv_list).unwrap()
+    /// fn inv_list_to_owned_codepointinversionlist(inv_list: &[u32]) -> CodePointInversionList {
+    ///     CodePointInversionList::clone_from_inversion_list_slice(inv_list).unwrap()
     /// }
     ///
     /// let bmp_list: [u32; 2] = [0x0, 0x10000];
     /// let smp_list: Vec<u32> = vec![0x10000, 0x20000];
     /// let sip_list: &[u32] = &vec![0x20000, 0x30000];
     ///
     /// let inv_lists: [&[u32]; 3] = [&bmp_list, &smp_list, sip_list];
-    /// let codepointsets: Vec<CodePointSet> = inv_lists
+    /// let codepointinversionlists: Vec<CodePointInversionList> = inv_lists
     ///     .iter()
-    ///     .map(|il| inv_list_to_owned_codepointset(il))
+    ///     .map(|il| inv_list_to_owned_codepointinversionlist(il))
     ///     .collect();
     ///
-    /// let bmp = &codepointsets.get(0).unwrap();
+    /// let bmp = &codepointinversionlists.get(0).unwrap();
     /// assert!(bmp.contains_u32(0xFFFF));
     /// assert!(!bmp.contains_u32(0x10000));
     ///
-    /// assert!(!&codepointsets.iter().any(|set| set.contains_u32(0x40000)));
+    /// assert!(!&codepointinversionlists.iter().any(|set| set.contains_u32(0x40000)));
     /// ```
     pub fn clone_from_inversion_list_slice(inv_list: &[u32]) -> Result<Self, CodePointSetError> {
         let inv_list_zv: ZeroVec<u32> = ZeroVec::alloc_from_slice(inv_list);
-        CodePointSet::from_inversion_list(inv_list_zv)
+        CodePointInversionList::from_inversion_list(inv_list_zv)
     }
 
-    /// Returns an owned inversion list representing the current [`CodePointSet`]
+    /// Returns an owned inversion list representing the current [`CodePointInversionList`]
     pub fn get_inversion_list(&self) -> Vec<u32> {
```

Comment:
Suggestion: name this `get_inversion_list_vec()` since the type is now called an "inversion list" too

Reply:
Done.